### PR TITLE
PMM-5257 Add ProxySQL socket conn.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-5257

ref:
 - https://github.com/percona/pmm/pull/511
 - https://github.com/percona/pmm-managed/pull/370
 - https://github.com/percona/pmm-admin/pull/101
 - https://github.com/Percona-Lab/pmm-api-tests/pull/92

FB: https://github.com/Percona-Lab/pmm-submodules/pull/815

### CODE REVIEW

**Comment by AlekSi** [comment]
There is merge conflict now.

**Comment by askomorokhov** [comment]
@artemgavrilov , could you please suggest how to pass this test https://github.com/percona/pmm-managed/blob/PMM-2.0/services/checks/checks_test.go#L34-L48

```
--- FAIL: TestDownloadChecks (0.30s)
1081    --- FAIL: TestDownloadChecks/normal (0.30s)
1082        checks_test.go:45: 
1083            	Error Trace:	checks_test.go:45
1084            	Error:      	Received unexpected error:
1085            	            	rpc error: code = Unimplemented desc = unknown service percona.platform.checked.v1beta1.CheckedAPI
1086            	            	failed to request checks service
1087            	            	github.com/percona/pmm-managed/services/checks.(*Service).downloadChecks
1088            	            		/root/go/src/github.com/percona/pmm-managed/services/checks/checks.go:463
1089            	            	github.com/percona/pmm-managed/services/checks.TestDownloadChecks.func1
1090            	            		/root/go/src/github.com/percona/pmm-managed/services/checks/checks_test.go:44
1091            	            	testing.tRunner
1092            	            		/usr/local/go/src/testing/testing.go:909
1093            	            	runtime.goexit
1094            	            		/usr/local/go/src/runtime/asm_amd64.s:1357
1095            	Test:       	TestDownloadChecks/normal
1096FAIL
```

**Comment by artemgavrilov** [comment]
@askomorokhov It's broken on SAAS side. Working on fix.

**Comment by artemgavrilov** [comment]
@askomorokhov that PR should fix the test, but we have another problem that will be solved only tomorrow. I can't do it myself 😕.

https://github.com/percona/pmm-managed/pull/378

**Comment by AlekSi** [comment]
SaaS side is fixed

File: models/database.go [hunk 1]
```diff
@@ -263,13 +263,23 @@ var databaseSchema = [][]string{
 
 	13: {
 		`ALTER TABLE services
-			DROP CONSTRAINT address_socket_check`,
+			DROP CONSTRAINT address_socket_check;`,
```
Comment (AlekSi) [inline models/database.go:266]: Not sure why we suddenly started to add `;`, but they are not required. I think it is better to remove them in all migrations for consistency.

Comment (askomorokhov) [reply models/database.go]: Removed.

**Pull request merged**